### ISSUE TITLE
Can't update to last version (^2.0.0-rc.15)

### ISSUE THREAD

**Issue by mishokupatadze** [bootstrap-vue/bootstrap-vue#2863]
![image](https://user-images.githubusercontent.com/46668694/54528542-991c0200-497d-11e9-9c16-2b5217f4bdb0.png)

Get this error after updating library to the last version(^2.0.0-rc.15). Whenever I downgrade to previous version(^2.0.0-rc.11) all good again. Also, I notified that application behind this error screen works fine.

P.S. I have checked and mentioned file exists.

OS: windows 10 (But the same happens on IOS)
Browser: Google Chrome 72.0.3626.121 (same result for all browsers)

**Comment by tmorehouse** [comment]
Have you tried removing `yarn.lock` and re-installing BV?

**Comment by stelgenhof** [comment]
I encountered a similar issue. Doing a `yarn upgrade` solved it for me.

**Comment by mishokupatadze** [comment]
@stelgenhof Thanks for the feedback! I have tried to yarn upgrade but no solution. Still the same problem.

**Comment by stelgenhof** [comment]
@mishokupatadze If you have changed your package.json file, you need to run `yarn upgrade` as that will upgrade all dependencies to the latest versions as defined in your package.json and subsequently will update the yarn.lock file.

Running `yarn install` will just install the dependencies according to the yarn.lock file (if present). 

After running either `yarn upgrade` or `yarn install`, you need to recompile your code.

TLDR:
1. `yarn install` / `yarn upgrade`
2. compile (e.g. `yarn dev`)

That is what I always do, and in most situations solve any issues. 

Cheers! Sacha

**Comment by mishokupatadze** [comment]
Thanks for feedback @tmorehouse and @stelgenhof ! We don't use yarn, just plain npm. Have tried to remove package-lock file and all node modules, change version and update. Still, fail and the same error :/
